charts/kubefed/charts/controllermanager/crds/crds.yaml

```diff
@@ -1,12 +1,7 @@
-{{ if (or (or (not .Values.global.scope) (eq .Values.global.scope "Cluster")) (not (.Capabilities.APIVersions.Has "core.kubefed.io/v1beta1"))) }}
```

Comment:
I may be out of date here, but dont we still care if certain deployments are scoped "cluster" or not?

Reply:
IIRC, with Helm3 and the `crds` folder, they don't support these Go template expressions. Instead if the user wants to skip the installation of the crds, they should run the `helm upgrade` command with the `--skip-crds` argument. <= This might be useful to have in the README.md

Reply:
indeed, there is no templating in CRDs anymore, starting from Helm v3. I've added a note regarding --skip-crds, for whatever reasons a user want this.